webapp/app/index.html

```diff
@@ -67,10 +67,10 @@
         <script src="js/services/classifications.js"></script>
         <script src="js/services/jobfilters.js"></script>
         <script src="js/services/pinboard.js"></script>
-        <script src="js/services/resultsets.js"></script>
         <script src="js/services/treestatus.js"></script>
         <!-- Model services -->
-        <script src="js/models/resultsets.js"></script>
+        <script src="js/models/resultset.js"></script>
+        <script src="js/models/resultset_store.js"></script>
```

Comment:
the file is referenced here as ``resultset_store.js`` but is actually ``resultsets_store.js`` (plural) as a filename.